tissuumaps/static/js/utils/regionUtils.js

```diff
@@ -1798,24 +1798,44 @@ regionUtils.JSONToRegions= function(filepath){
         if (path != null) {
             filepath = path + "/" + filepath;
         }
-        fetch(filepath)
-        .then((response) => {
-            return response.json();
-        })
-        .then((regionsobj) => {
-            regionUtils.JSONValToRegions(regionsobj);
-        });
+        if (filepath.includes(".pbf")) {
```

Comment:
Is there any reason for not checking with `.endsWith`? Also, how about uppercase?

Reply:
No. Fixed it now, and also added conversion to lowercase.

Reply:
> Also, how about uppercase?

That's interesting, I don't think the flask code supports uppercase extensions. Should we fix this everywhere? Can be quite a lot of changes.

Reply:
After thinking about it a bit, let's not go there yet. We support lowercase file extensions for now within the GUI.
We keep this for later, as flask is url based and difference between lowercase/uppercase urls can be tricky to handle in some cases.